AdyenUI/UI/Form/FormItemViewBuilder.swift

```diff
@@ -9,28 +9,28 @@ import Foundation
 /// Builds different types of `FormItemView's`  from the corresponding concrete `FormItem`.
 @_spi(AdyenInternal)
 public struct FormItemViewBuilder {
-    
-    /// The theme to use for building views. Optional during migration.
-    package let theme: AdyenTheme?
-    
+
+    /// The theme to use for building views.
+    package let theme: AdyenTheme
+
     /// Initializes the form item view builder.
-    /// - Parameter theme: The theme to use for building views. If nil, views will use item.style.
+    /// - Parameter theme: The theme to use for building views. Defaults to `.default` if nil.
     package init(theme: AdyenTheme? = nil) {
```

Comment:
I see theme is not optional any more then we can add it as default parameter in the init

Reply:
Fair, this is indeed one step forward. I tried to make this initializer easily breakable to quickly see what's left but this can be done with a non-optional theme as well!